website/source/docs/state/remote/consul.html.md

```diff
@@ -47,3 +47,5 @@ The following configuration options / environment variables are supported:
  * `datacenter` - (Optional) The datacenter to use. Defaults to that of the agent.
  * `http_auth` / `CONSUL_HTTP_AUTH` - (Optional) HTTP Basic Authentication credentials to be used when
    communicating with Consul, in the format of either `user` or `user:pass`.
+ * `gzip_state` - (Optional) Specifies whether the state should be stored gzipped as opposed to
```

Comment:
Sorry that this is a bit of a nitpick... feels like this should be called just `gzip` since it's implied that we're talking about state by the context here.

Since I'm already nit-picking, here's another: since remote state properties don't natively support booleans (they're always strings) I think it would be helpful to be specific here about what strings are required. So maybe something like this:

```
* `gzip` - (Optional) `true` to compress the state data using gzip, or `false` (the default) to leave it uncompressed.
```